applications/Chat/examples/generate_prompt_dataset.sh

```diff
@@ -0,0 +1,4 @@
+python generate_prompt_dataset.py \
```

Comment:
Add this help message to `argparse` by adding `help="xxxxx"`. This shell script cannot be run, so I think it can be removed. Simple description in README and cli help message are enough.

Reply:
Now done.